packages/node/src/methods/app-instance/install/operation.ts

```diff
@@ -21,8 +16,9 @@ export async function install(
   initiatingAddress: string,
   respondingAddress: string,
   params: Node.InstallParams
-): Promise<AppInstanceInfo> {
+): Promise<ProposedAppInstanceInfo> {
```

Comment:
Why is installing an AppInstance returning a promise for a Proposed AppInstance?